libraries/core/src/main/kotlin/com/bumble/appyx/core/plugin/Plugins.kt

```diff
@@ -23,6 +23,11 @@ interface NodeLifecycleAware : Plugin {
     fun onCreate(lifecycle: Lifecycle) {}
 }
 
+interface GlobalNodeLifecycleAware : Plugin {
```

Comment:
Can be implemented as combination of `NodeAware` and `NodeLifecycleAware`?

Reply:
As discussed in in person this won't work for the usecase.